packages/http-server/src/server.ts

```diff
@@ -41,6 +41,10 @@ export const createServer = (operations: IHttpOperation[], opts: IPrismHttpServe
   });
 
   server.addHook('onSend', (request, reply, payload, done) => {
+    if (reply.getHeader('content-type') === 'application/json') {
```

Comment:
What is the reasoning for this? If the api designer has defined the header as `application/json`why do we want to force the utf8 charset on there?

Reply:
What about support for `application/{dataType}+json` or `application/json; charset=cp1250`?

Source: https://tools.ietf.org/html/rfc6839

Reply:
@philsturgeon, @StefanDywersant, `charset=utf-8` was already enforced by `fastify` with its default `application/json` serializer. We sometimes returned `application/json` without `charset=utf-8` because... the response was wrongly serialized by some other serializer (like instead of a serializer for JSON, the data was serialized with a serializer for XML). `application/{whatever}+json` is here https://github.com/stoplightio/prism/blob/master/packages/http-server/src/serializers.ts#L9
haven't found other charset than utf-8 logic in the code

Reply:
> What about support for application/{dataType}+json

This scenario is already supported in the serialiser. I would not bother about the `charset` — it is actually something that is inferred by the browser most of the times. I am quite sure there was also a proposal to remove it completely from the content type header.